pyramid_swagger/tween.py

```diff
@@ -456,9 +482,24 @@ def swaggerize_response(response, **kwargs):
     Delegate handling the Swagger concerns of the response to bravado-core.
 
     :type response: :class:`pyramid.response.Response`
+    :type op: :class:`bravado_core.operation.Operation`
     """
-    # TODO: validate, marshal, and transform the response object
-    log.warn('TODO: Implement swaggerize_response()')
+    op = kwargs['op']
+    try:
+        response_spec = get_response_spec(response.status_int, op)
+    except SwaggerMappingError:
+        error = ResponseValidationError(
+            "Could not find a matching Swagger response for {0} request {1}"
+            "with http_status {2}.".format(
+                op.http_method.upper(), op.path_name, response.status_code))
+
+        # Workaround for "W602 deprecated form of raising exception"
+        # See https://github.com/jcrocholl/pep8/issues/34
```

Comment:
`# noqa` didn't want to work for some reason. Had to use `flake8: noqa` to get it to actually ignore the warning.